### DISCUSSION TITLE
NotBocchi 的网站！&绳网官方DC频道

### DISCUSSION THREAD

**Discussion by inkluv** [share121/inter-knot#3061] [其他]
[银狼.ink](http://xn--n6xt83e.ink/)
### 欢迎访问！
内有资源分享盘，不定时上传RVC模型、绳网DC频道逆天发言截图等文件

有RVC模型需求欢迎来DC频道找我！
——————————————
[加入绳网DC频道，成为正式绳匠！](https://discord.gg/interknot)

**Comment by lllliufeng** [comment]
你说得对，但是我打不开

**Reply by inkluv** [reply]
抱歉我链接搞错了

**Reply by inkluv** [reply]
再试试

**Reply by lllliufeng** [reply]
有了o(*￣▽￣*)o

**Comment by 1ctrl-cv** [comment]
坏了，discord人机验证挂梯也打不开（

**Comment by qmtqyh** [comment]
哇楼主头像跟我qq头像撞了耶
![Uploading 1C90B4CE6F3F13BB5D84B4F791F4A086.png…]()
